examples/specs/window_transform_movie_mean_difference.vl.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
-    "description": "Bar graph showing how each movie is different from the average movie rating",
+    "description": "Bar graph showing how each of the 'really good' movies is different from the average movie rating. Really good is defined to differ from the average movie rating by more than 2.5 points.",
```

Comment:
This plot would benefit from another layer of rule that show global average rating (and perhaps another layer of text to label the average)